crates/meilisearch-types/src/batch_view.rs

```diff
@@ -35,7 +35,7 @@ impl BatchView {
             duration: batch.finished_at.map(|finished_at| finished_at - batch.started_at),
             started_at: batch.started_at,
             finished_at: batch.finished_at,
-            batcher_stopped_because: batch.stop_reason.clone(),
+            batch_creation_complete: batch.stop_reason.clone(),
```

Comment:
Do we want to keep `stop_reason` internally?

Reply:
yes I find the stop reason clearer for internal purposes.